### ISSUE TITLE
ensembl proteins point to  grch38 transcripts

### ISSUE THREAD

**Issue by bwalsh** [bmeg/bmeg-etl#241]
```
cat import.report | grep "missing node Transcript"   | awk '{print $NF}' | sort | uniq  | wc -l
>> 21500 
```

This is due to the fact that `transform/ensembl-protein/ensembl-protein-transform.py` harvests from `ftp://ftp.ensembl.org/pub/release-93/json/homo_sapiens/homo_sapiens.json` this file is based on **grch38**

There does not seem to have a identical json file published for  grcg37

Alternatives:
* Reverse engineer from a new data source:  ftp://ftp.ensembl.org/pub/grch37/release-94/rdf/homo_sapiens.ttl.gz
  * this is a heavy weight solution as the file is huge, complicated and apparently undocumented
* Complete the graph by adding the missing transcriptions. By harvesting from:

```
    url = 'https://rest.ensembl.org/lookup/id/{}?content-type=application/json;expand=1'.format(ensembl_transcript_id)
    r = requests.get(url, timeout=20)
    response = r.json()
    # {
    #   "db_type":"core",
    #   "start":55287849,
    #   "Parent":"ENSG00000115355",
    #   "end":55296455,
    #   "object_type":"Transcript",
    #   "logic_name":"havana",
    #   "assembly_name":"GRCh38",
    #   "species":"homo_sapiens",
    #   "biotype":"protein_coding",
    #   "seq_region_name":"2",
    #   "id":"ENST00000647547",
    #   "display_name":"CCDC88A-268",
    #   "source":"havana",
    #   "strand":-1,
    #   "is_canonical":0,
    #   "version":1
    # }
```

**Comment by bwalsh** [comment]
@kellrott @adamstruck - can you review?

**Comment by adamstruck** [comment]
Fixed in d023a1ff979636cbd141c7bfaa407a79199e1525
